src/waveform/renderers/waveformmark.cpp

```diff
@@ -8,7 +8,8 @@
 #include "widget/wskincolor.h"
 
 WaveformMark::WaveformMark()
-    : m_pPointCos(nullptr) {
+    : m_pPointCos(nullptr),
```

Comment:
It would be a good opportunity to replace the plain ControlProxy* pointer by std::unique_ptr<ControlProxy> that does not need to be initialized with nullptr and does not need to be deleted either. Then the destructor of WaveformMark becomes obsolete.